pandas/tests/arrays/categorical/test_analytics.py

```diff
@@ -354,7 +354,7 @@ def test_validate_inplace_raises(self, value):
         with pytest.raises(ValueError, match=msg):
             cat.remove_categories(removals=["D", "E", "F"], inplace=value)
 
-        with pytest.raises(ValueError, match=msg):
+        with tm.assert_produces_warning(FutureWarning):
             cat.remove_unused_categories(inplace=value)
```

Comment:
comment inside this block about why we're getting a FutureWarning.

im guessing this actually needs both the raises and the produces_warning, ie.

```
with pytest.raises(ValueError, match=msg):
    with tm.assert_produces_warning(FutureWarning):
        # GH#37918 inplace kwarg deprecated
        cat.remove_unused_categories(inplace=value)
```

Reply:
Ok